Gopkg.toml

```diff
@@ -18,20 +18,42 @@ required = [
 
 [prune]
   go-tests = true
+  unused-packages = true
+  non-go = true
 
-
-# STANZAS BELOW ARE GENERATED AND MAY BE WRITTEN - DO NOT MODIFY BELOW THIS LINE.
+# For dependency below: Refer to issue https://github.com/golang/dep/issues/1799
+[[override]]
+name = "gopkg.in/fsnotify.v1"
+source = "https://github.com/fsnotify/fsnotify.git"
+version="v1.4.7"
 
 [[constraint]]
+  name = "github.com/knative/serving"
+  version = "v0.5.0"
```

Comment:
Why are we locking this version w/ a constraint?

Reply:
I thought it is good to lock the version, since `required` may find the latest version and your current code may break, maybe I do not fully understand how `required` works.